src/cantools/database/can/c_source.py

```diff
@@ -804,6 +805,7 @@ class CodeGenMessage:
 
     def __init__(self, message: "Message") -> None:
         self.message = message
+        self.name = message.name
```

Comment:
This should be a `@property`

Reply:
could you please submit a proposal? thanks!